messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/pooled/WorkPackage.java

```diff
@@ -418,17 +418,17 @@ public void extendClaimIfThresholdIsMet() {
                     unitOfWorkFactory
                             .create()
                             .executeWithResult(context -> {
-                                tokenStore.extendClaim(name, segment.getSegmentId());
+                                tokenStore.extendClaim(name, segment.getSegmentId(), null);
                                 return emptyCompletedFuture();
```

Comment:
```suggestion
                                tokenStore.extendClaim(name, segment.getSegmentId(), context);
return emptyCompletedFuture();
```

We should pass the `ProcessingContext` here.